pallets/registrar/src/benchmarks.rs

```diff
@@ -116,32 +116,16 @@ mod benchmarks {
 
     #[benchmark]
     fn register(x: Linear<100, 3_000_000>, z: Linear<1, 10>) {
-        let y = T::MaxLengthParaIds::get();
         let storage = max_size_genesis_data::<T>(z, x);
 
-        for i in 1..y {
```

Comment:
This was removed because we removed `y` right?

Reply:
it's mainly because it made sense before because the `pendingVerification` storage was a vector and not a mapping. And therefore, the cost of inserting a new one depended on the number of chains already existing.

@tmpolaczyk changed this storage item from a vector to a mapping. And therefore now it does not matter how many chains you have `pendingVerification`, so this parameter can be removed. And even better, speeds up benchmarks a lot